UpdateService/handler.ts

```diff
@@ -148,9 +150,22 @@ const updateServiceTask = (
  * Handles requests for updating a service by given serviceId and a Service Payload.
  */
 export function UpdateServiceHandler(
+  telemetryClient: ReturnType<typeof initAppInsights>,
   apiClient: APIClient
 ): IUpdateServiceHandler {
   return (_, apiAuth, ___, userAttributes, serviceId, servicePayload) => {
+    const trackUpdateServiceVisibility = (
```

Comment:
there's no need to wrap `telemetryClient.trackEvent` into a closure, just use it inline

Reply:
Fixed in [6aae3a7](https://github.com/pagopa/io-functions-services/pull/87/commits/6aae3a75463140ef02f6ed99c95de6ba45e9d011)